packages/cli/src/cmds/preview/commonUtils.ts

```diff
@@ -93,7 +93,7 @@ export function createTaskStartCb(
             color: Colors.BRIGHT_GREEN,
           },
         ];
-        cliLogger.necessaryLog(LogLevel.Info, getColorizedString(message));
+        await cliLogger.log(LogLevel.Debug, getColorizedString(message));
```

Comment:
How will this impact `npm run dev:teamsfx` tasks?

Reply:
The location of detail log is hidden for INFO level.
When users debug `preview`,  they can set DEBUG level to get the location of detail log for tasks.

Reply:
I prefer showing the localtion of log for INFO level. Otherwise it is not convenient for user to see the service status and troubleshot.

Reply:
Well.. It's a little inconvenient.
But, service status can be displayed by a progress bar. I just think it's normal for users to use `teamsfx preview --debug` for troubleshooting.  
And we have shown how to use --debug.
![image](https://user-images.githubusercontent.com/12523371/157797608-f6c777f0-28a0-4234-9d41-18efbe0cd63f.png)

Reply:
In my understanding, `--debug` is to debug the CLI, for example if some unexpected error happens, CLI users can provide these debug information then CLI developers can know what happens. However, the service status is necessary for users to debug their apps.

Reply:
@kuojianlu What is your concern?
Users can get location of detail log with `teamsfx preview --debug`, as shown below:
![image](https://user-images.githubusercontent.com/12523371/157812979-b0cda825-5afd-4663-a87e-9baae85cf9fd.png)

![image](https://user-images.githubusercontent.com/12523371/157812926-65c8ebd1-abd8-482b-9db5-ad64eeb6460a.png)

Reply:
Another scenario is that users may also want to see their output contents in codes. Suppose users write many `console.log(xxx)` codes, and they want to know which outputs are printed when debugging the app.

Reply:
In that scenario, we do need give user the location of services log. 
Added.